src/lib/types/Events.ts

```diff
@@ -198,6 +198,7 @@ export interface MessageCommandDeniedPayload extends IMessageCommandPayload {
 export interface MessageCommandAcceptedPayload extends IMessageCommandPayload {
 	parameters: string;
 	context: MessageCommand.RunContext;
+	duration?: number;
```

Comment:
You typed this as possibly undefined but it's always defined..

Reply:
Fixed it. Turns out the problem was a bit deeper in fact because of how the interfaces extend one another.